modules/gdscript/gdscript_warning.cpp

```diff
@@ -76,18 +76,9 @@ String GDScriptWarning::get_message() const {
 		case STANDALONE_EXPRESSION:
 			return "Standalone expression (the line has no effect).";
 		case STANDALONE_TERNARY:
-			return "Standalone ternary conditional operator: the return value is being discarded.";
+			return "Standalone ternary operator: the return value is being discarded.";
```

Comment:
I'm guessing that `STANDALONE_TERNARY` was probably added because of the "(the line has no effect)" in `STANDALONE_EXPRESSION` message. But in fact, these warnings are not much different. The static analyzer is currently not very good at detecting side effects, so `STANDALONE_EXPRESSION` may be produced for expressions with side effects. Could we reformulate the message so that it is not misleading?

Anyway, we already have `STANDALONE_TERNARY`, so it probably makes sense to make the warning work rather than deprecating it and producing `STANDALONE_EXPRESSION` instead?

Reply:
> The static analyzer is currently not very good at detecting side effects, so STANDALONE_EXPRESSION may be produced for expressions with side effects.

Yes, we're thinking about getters that introduce side-effects.

![image](https://github.com/godotengine/godot/assets/270928/51bad084-fdf1-4f9d-9d32-68f2d83def1e)

We could modify the `STANDALONE_EXPRESSION` to remove the parenthesis `(the line has no effect)`.

If we had error numbers, we could add a description to the error. Food for thought.

> Anyway, we already have STANDALONE_TERNARY, so it probably makes sense to make the warning work rather than deprecating it and producing STANDALONE_EXPRESSION instead?

So, as @dalexeev said in the meeting:
> Danil Alexeev says:STANDALONE_TERNARY is not currently implemented. This was just migrated from 3.x with no implementation. 

Yes, we could make `STANDALONE_TERNARY` work, if it's not too difficult to implement.

Reply:
@dalexeev in the meeting:
> We have a choice: either implement this (already done in the PR) or deprecate it and produce STANDALONE_EXPRESSION instead. 

We think it's ok then! Great!